DEPS

```diff
@@ -27,16 +27,37 @@ vars = {
   # When updating the Dart revision, ensure that all entries that are
   # dependencies of Dart are also updated to match the entries in the
   # Dart SDK's DEPS file for that revision of Dart. The DEPS file for
-  # Dart is: https://github.com/dart-lang/sdk/blob/master/DEPS
-  'dart_revision': '65a5707189dec6c5e0174cbc821acfb90f0c9b3f',
-  # Dart calls the next one "boringssl_gen_rev"
-  'dart_boringssl_gen_revision': '753224969dbe43dad29343146529727b5066c0f3',
-  # Dart calls the next one "boringssl_rev"
-  'dart_boringssl_revision': 'd519bf6be0b447fb80fbc539d4bff4479b5482a2',
-  # Dart calls the next one "observatory_pub_packages_rev"
-  'dart_observatory_packages_revision': '26aad88f1c1915d39bbcbff3cad589e2402fdcf1',
-  # Dart calls the next one "root_certificates_rev"
-  'dart_root_certificates_revision': 'a4c7c6f23a664a37bc1b6f15a819e3f2a292791a',
+  # Dart is: https://github.com/dart-lang/sdk/blob/master/DEPS.
+  # You can use //dart/tools/create_updated_fluter_deps.py to produce
```

Comment:
Well, another source of hesitation is that DEPS file even though in python seems intrinsically handcrafted with comments, formatting and some such, :-) , so automatically updating it without disturbing engineers sense of beauty might require some effort.
Something along the lines where we first read DEPS file as python, figure out updated list of updated versions, then re-read DEPS file as text looking for some comments indicating start and end of the dart dependencies versions section, replace that textually. Doable, but perhaps a little complicated for this cl, so we could do it later.